simpleSequential/test_isIntegerListEmpty.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+from isIntegerListEmpty import isIntegerListEmpty
+
+class TestIsIntegerListEmpty(unittest.TestCase):
```

Comment:
Test name is inconsistent and not on point. The function focus on the input list is empty or not. The test name is two long and I cannot understand what is the test case when I just read it.

Please use:

1/ emptyListInput
2/ nonEmptyListInput

Short, and on point.